docs/source/deploy_groups.rst

```diff
@@ -2,17 +2,15 @@
 Deploy Groups
 =============
 
-What are deploy groups?:
+What are deploy groups?
 ========================
 
-Deploy groups are a way to allow tron jobs and marathon services to share docker containers, git branches and deployment steps. Deploy groups also shorten your jenkins pipeline and make it run faster (or save you time if you manually mark your changes for deployment)!
+A deploy group is a group of Paasta instances that will be deployed together.
```

Comment:
I wonder if we should add something more explicit about how they're a way to control how your service will rollout to production in this summary? we somewhat get to that later, but it might be useful to have that upfront as well (in case people aren't interested in the rest of the details)

Reply:
I think I addressed this in https://github.com/Yelp/paasta/pull/3115/commits/78826f5e3e40904fb8ae40f697858f4ce7783db2#diff-b6f4164b8cadd541b5bb0576e6cc42aea374be0b6d03c12d6c181de924be6583R9